### ISSUE TITLE
dev container broken due to deletion of setup.py in simtools

### ISSUE THREAD

**Issue by VictorBarbosaMartins** [gammasim/containers#51]
The deletion of `setup.py` in [https://github.com/gammasim/simtools/pull/522](url) actually broke the dev container (see message below when trying to build it):

```
 => CACHED [stage-1 4/8] RUN apt-get update && apt-get install -y     bc   0.0s
 => CACHED [stage-1 5/8] RUN wget --quiet https://bootstrap.pypa.io/get-p  0.0s
 => ERROR [stage-1 6/8] RUN wget --quiet https://raw.githubusercontent.co  0.7s
```

The command running with `--quiet` hid the error, which is the missing file:

```
wget https://raw.githubusercontent.com/gammasim/simtools/main/setup.py
--2023-08-14 14:22:45--  https://raw.githubusercontent.com/gammasim/simtools/main/setup.py
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.110.133, 185.199.111.133, 185.199.108.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.110.133|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2023-08-14 14:22:46 ERROR 404: Not Found.
```

**Comment by VictorBarbosaMartins** [comment]
Sorry, did not see that #49 is already addressing this.

**Comment by GernotMaier** [comment]
Fixed in PR #49.
